### ISSUE TITLE
Module minimum perl changed to latest 5.36

### ISSUE THREAD

**Issue by stepht** [drclaw1394/perl-template-plex#1]
Hi
I installed a previous version of Hustle::Table to look at it, both on cygwin and strawberry....
o maybe installed first a version of Template::Plex

but you bumped the version to latest and with exp. features

stock cygperl is stalled at 5.32 sb perl's too 
windows-git perl at 5.34
in my library code I expect at least 5.20 often 5.26

I think it's fair for users to put clearly in the docs that you are going to target only latest perl

hth
cheers

**Comment by drclaw1394** [comment]
Hi stepht,

I did a bunch of feature improvements to Template::Plex, which I wanted to focus on newer versions of perl. I realised after releasing it that Hustle::Table would break on earlier  perl versions. I'll looking at fixing this asap. 

If I have time later today/tonight I'll try and get the min perl version back to 5.24 or 5.26,  what ever it was previously.

Thanks for the feed back regarding documentation. I'll take that on board.
Ruben
